### PR TITLE
Modify the weights in compute_loss

### PR DESCRIPTION
**Pull request by shanmo**
*Description of changes:*

In [neural style](https://github.com/d2l-ai/d2l-en/blob/master/chapter_computer-vision/neural-style.md) the losses are added by `l = sum(10 * styles_l + contents_l + [tv_l])`, but the text does not seem to provide any justification about why an additional multiplication of `10` is applied to `styles_l`. I checked the Eq. 7 in [original paper](https://arxiv.org/pdf/1508.06576.pdf), the [implementation in keras](https://github.com/keras-team/keras-io/blob/f7da9514bec1b1be335c5610fbe4b31eb4c71de0/examples/generative/neural_style_transfer.py#L214) and [implementation in tensorflow](https://github.com/tensorflow/docs/blob/master/site/en/tutorials/generative/style_transfer.ipynb), but could not find why an additional multiplication should be applied to style loss either. Perhaps it is less confusing to just increase the `style_weight` by 10 times.

The plot for original loss plots `styles_l` but in fact `10 * styles_l` is used, and thus the styles_l loss in the plot is smaller:
image

The updated version directly plots the actual `styles_l`:
image

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-2029/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-2029/

**Comment by AnirudhDagar** [comment]
@d2l-bot please rebuild!
